src/standalone/Vegetation/canopy_boundary_fluxes.jl

```diff
@@ -303,7 +336,8 @@ end
         gustiness::FT,
         z_0m::FT,
         z_0b::FT,
-        earth_param_set::EP,
+        earth_param_set::EP;
+        compute_momentum_fluxes = false,
```

Comment:
we might use a different name here - momentum_fluxes are always computed, but in some cases they are not returned

Reply:
Good point! I'll change it to `return_momentum_fluxes`